### PR TITLE
V2 alpha css

### PR DESCRIPTION
**Pull request by NemTechCN**
- logos changes 
- new fonts (supported by branding guidelines)
- menu colors
- account balance panel image background
- wallet pages CSS fixes for symbol (list of wallets on left side)

### CODE REVIEW

File: src/components/MosaicAttachmentDisplay/MosaicAttachmentDisplay.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@
               {{ m.name }}
             </span>
             <span class="mosaic_amount overflow_ellipsis">
-              <MosaicAmountDisplay :id="m.id" :relative-amount="m.amount" :absolute="absolute" :size="'normal'" />
+              <MosaicAmountDisplay :id="m.id" :relative-amount="m.amount" :absolute="absolute" :size="'smaller'" />
```
Comment (evias) [inline src/components/MosaicAttachmentDisplay/MosaicAttachmentDisplay.vue]: please keep `normal` size here.

Comment (NemTechCN) [reply src/components/MosaicAttachmentDisplay/MosaicAttachmentDisplay.vue]: reverted size to "normal"

File: src/language/en-US.json [hunk 2]
```diff
@@ -787,8 +790,9 @@
   "Import_Mnemonic_phrase_directly_to_make_an_account": "Import Mnemonic passphrase to create an account",
   "Access_your_trezor_wallet_to_make_trezor_account": "Access your Trezor wallet",
   "Access_your_ledger_wallet_to_make_ledge_account": "Access your Ledger wallet",
-  "This_program_is_free": "This software is a desktop wallet for Catapult. Easy to use and lets you explore the features of distributed ledger technology powered by Catapult.",
-  "Nem_Catapult_Welcome_you_nice": "Welcome to Symbol",
+  "This_program_is_free": "This program is designed based on Symbol blockchain ,an open source Symbol digital asset management system. Access the Symbol network, create account manage assets and more…",
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/language/en-US.json]: this words comes from "brand guidlines" file, but less one  wihtespace

File: src/views/pages/accounts/create-account/show-mnemonic/ShowMnemonic.less [hunk 3]
```diff
@@ -46,12 +46,12 @@
...
     span {
       display: inline-block;
-      background-color: #8fdad5;
+      background-color: #6401ee;
       padding: 5px 10px;
       border-radius: 4px;
       margin: 5px;
-      font-size: 26px;
-      color: white;
+      font-size: 18px;
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/views/pages/accounts/create-account/show-mnemonic/ShowMnemonic.less]: got it, thanks

File: src/language/en-US.json [hunk 4]
```diff
@@ -775,9 +778,9 @@
   "Input_account_public_key": "Enter an account public key",
   "to_or_action": "To/Action",
   "account_restriction_address": "Account address restriction",
-  "login_account": "Login account",
+  "login_account": "Login",
   "login": "Login",
-  "create_a_new_account": "Create a new account",
+  "create_a_new_account": "Have no Account",
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/language/en-US.json]: In the new registration process, no account is created, only the mnemonic is created, and then the account is set up.  "Have no Account"， Is there a syntax error？

**Comment by decentraliser** [comment]
Yes, I don't think dropping "have no account" like that is correct


Le ven. 21 févr. 2020 à 22:18, NemTechCN <notifications@github.com> a
écrit :

File: src/language/en-US.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (decentraliser) [reply src/language/en-US.json]: yeah it does not sound right to dro "have no account" like that

File: src/language/en-US.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (decentraliser) [reply src/language/en-US.json]: oh waw just seen that, ok

please update the punctuation, will bring that up in the slack

File: src/components/TableDisplay/TableDisplay.less [hunk 5]
```diff
@@ -1,10 +1,9 @@
 .mosaic-columns {
-  grid-template-columns: repeat(5, 13.5%) repeat(4, 7%) 4.5%;
+  grid-template-columns: repeat(4, 9.5%) repeat(4, 13%) 12%;
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/components/TableDisplay/TableDisplay.less]: thanks

File: src/language/en-US.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/language/en-US.json]: like this "more ...", the punctuation means that add one whitespace ?

File: src/language/en-US.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/language/en-US.json]: need help ... ， if not use "Create a new account"

**Comment by NemTechCN** [comment]
rebase it and resolve above comment issues and some conflicts. @evias @decentraliser

**Comment by decentraliser** [comment]
I'm gonna take it over, please create a new branch for new pushes!

File: src/views/pages/accounts/create-account/finalize/Finalize.vue [hunk 6]
```diff
@@ -46,7 +46,7 @@
         </div>
       </div>
       <div class="finish-right">
-        <img src="@/views/resources/img/finish.png" alt>
+        <!-- <img src="@/views/resources/img/finish.png" alt> -->
```
Comment (decentraliser) [inline src/views/pages/accounts/create-account/finalize/Finalize.vue]: check

File: src/views/pages/accounts/LoginPage.less [hunk 7]
```diff
@@ -19,14 +19,22 @@
...
-  background-size: cover;
+  // background:linear-gradient(161deg,rgba(82,0,198,1) 0%,rgba(68,0,78,1) 100%);
+  background: linear-gradient(121deg, rgba(82, 0, 198, 1) 0%, rgba(78, 0, 167, 1) 100%);
+  opacity: 1;
   overflow: hidden;
   position: absolute;
   left: 0;
   top: 0;
   z-index: 999;
-  // opacity: .8;
```
Comment (decentraliser) [inline src/views/pages/accounts/LoginPage.less]: remove

File: src/views/pages/accounts/LoginPage.less [hunk 8]
```diff
@@ -19,14 +19,22 @@
 .login-account-wrapper {
   width: 100%;
   height: 100%;
-  background-image: linear-gradient(120deg, #5200c6ce, #44004edc);
-  background-size: cover;
+  // background:linear-gradient(161deg,rgba(82,0,198,1) 0%,rgba(68,0,78,1) 100%);
```
Comment (decentraliser) [inline src/views/pages/accounts/LoginPage.less]: remove

File: src/language/en-US.json [hunk 9]
```diff
@@ -787,8 +789,9 @@
   "Import_Mnemonic_phrase_directly_to_make_an_account": "Import Mnemonic passphrase to create an account",
   "Access_your_trezor_wallet_to_make_trezor_account": "Access your Trezor wallet",
   "Access_your_ledger_wallet_to_make_ledge_account": "Access your Ledger wallet",
-  "This_program_is_free": "This software is a desktop wallet for Catapult. Easy to use and lets you explore the features of distributed ledger technology powered by Catapult.",
-  "Nem_Catapult_Welcome_you_nice": "Welcome to Symbol",
+  "program_illustration": "This program is designed based on Symbol blockchain, an open source Symbol digital asset management system. Access the Symbol network, create account manage assets and more …",
```
Comment (decentraliser) [inline src/language/en-US.json]: punctuation

Comment (decentraliser) [inline src/language/en-US.json]: rename

File: src/components/WalletSelectorPanel/WalletSelectorPanel.vue [hunk 10]
```diff
@@ -2,18 +2,9 @@
...
-        <h1 class="section-title">
-          {{ $t('Wallet_management') }}
-        </h1>
-      </div>
-      <div class="wallet-switch-header-right-container">
-        <span class="back-up pointer">
-          {{ $t('backup_mnemonic') }}
-          <!-- @click="displayMnemonicDialog" -->
-        </span>
+        <h1 class="section-title">{{ $t('Wallet_management') }}</h1>
```
Comment (decentraliser) [inline src/components/WalletSelectorPanel/WalletSelectorPanel.vue]: check

File: src/components/WalletSelectorPanel/WalletSelectorPanel.less [hunk 11]
```diff
@@ -102,20 +106,46 @@
...
+      }
+    }
+
+    .wallet-switch-header-right-container {
+
+      img {
+        width: 18px;
+        height: 16px;
+        padding-right: 5px;
+        // margin-top: 5px;
```
Comment (decentraliser) [inline src/components/WalletSelectorPanel/WalletSelectorPanel.less]: remove

File: src/components/PageNavigator/PageNavigator.vue [hunk 12]
```diff
@@ -25,9 +25,6 @@
         <div class="navigator-icon-container">
           <Icon :type="'md-log-out'" class="navigator-icon" />
         </div>
-        <div class="navigator-text-container">
-          {{ currentAccount.values.get('accountName') }}
```
Comment (decentraliser) [inline src/components/PageNavigator/PageNavigator.vue]: check

File: src/components/NetworkStatisticsPanel/NetworkStatisticsPanel.vue [hunk 13]
```diff
@@ -5,38 +5,36 @@
...
         <div class="title">
-          <span class="title_txt">{{ $t('transactions') }}</span>
+          <span class="title_txt">{{ $t('speed') }}</span>
         </div>
-        <img src="@/views/resources/img/windowDashboardActive.png">
-        <span class="txt_info">
-          <AnimatedNumber v-model="countTransactions" />
+        <img src="@/views/resources/img/monitor/speed.png" />
+        <span class="txt_info speed">
+          <!-- <AnimatedNumber v-model="countTransactions" /> -->
```
Comment (decentraliser) [inline src/components/NetworkStatisticsPanel/NetworkStatisticsPanel.vue]: remove

File: src/components/NetworkStatisticsPanel/NetworkStatisticsPanel.vue [hunk 14]
```diff
@@ -5,38 +5,36 @@
...
+          15s
```
Comment (decentraliser) [inline src/components/NetworkStatisticsPanel/NetworkStatisticsPanel.vue]: no hardcoded val

File: src/components/NetworkStatisticsPanel/NetworkStatisticsPanel.vue [hunk 15]
```diff
@@ -5,38 +5,36 @@
         <div class="title">
           <span class="title_txt">{{ $t('accounts') }}</span>
         </div>
-        <img src="@/views/resources/img/monitor/network.png">
+        <img src="@/views/resources/img/monitor/point_amount.png" />
```
Comment (decentraliser) [inline src/components/NetworkStatisticsPanel/NetworkStatisticsPanel.vue]: rename

File: src/components/MnemonicVerification/MnemonicVerification.less [hunk 16]
```diff
@@ -80,6 +81,6 @@
   }
 
   .ivu-tag {
-    height: 26px;
+    height: auto;
```
Comment (decentraliser) [inline src/components/MnemonicVerification/MnemonicVerification.less]: check

File: src/components/AccountBalancesPanel/AccountBalancesPanel.less [hunk 17]
```diff
@@ -40,12 +41,14 @@
   padding: 22px 20px 0 21px;
   position: relative;
   background-size: cover;
+  box-shadow: 0 0.01rem 0.15rem 0.01rem #d8c5f3;
+  background: linear-gradient(161deg, rgba(82, 0, 198, 1) 0%, rgba(68, 0, 78, 1) 100%);
+  border-radius: 10px;
   // background-image: url("../../views/resources/img/monitor/dash-board/dashboardWalletBackground.png");
```
Comment (decentraliser) [inline src/components/AccountBalancesPanel/AccountBalancesPanel.less]: remove

**Pull request merged**